### ISSUE TITLE
wordpress.com adds tracking pixel (gif) that teaserImage pulls instead of post image

### ISSUE THREAD

**Issue by siwise** [sdepold/jquery-rss#115]
Hi,
Thanks for a great tool! 
As stated, wordpress.com add a crazy tracking pixel as an image for every post. There is no way to disable this "feature" on wordpress.com and all themes seem to enable this "feature". 

Is there any simple way of forcing teaserImage and teaserImageUrl to call the second or third image? Hoping this would get around this stupid tracking pixel issue. 

I have tried messing with different number combinations of this line but haven't found one that works. 

`teaserImageUrl: (function (entry) {
        try {
          return entry.content.match(/(<img.*?>)/gi)[0].match(/src="(.*?)"/)[1];
        }`

Thanks for your help!

**Comment by sdepold** [comment]
Can you paste feed url?
siwise <notifications@github.com> schrieb am Mi., 10. Aug. 2016 um 02:44:

**Comment by siwise** [comment]
Tried with https://asijnews.com/feed/ and https://asijnews.com/feed/?mrss=off

**Comment by sdepold** [comment]
For whatever reason wordpress is not embedding the images into the post description itself. Instead it puts it outside of the description as `media` tags. So what one would need to do is to go into the feedrapp and add support for them their. https://github.com/sdepold/feedrapp

**Comment by sdepold** [comment]
Here we go :) https://github.com/sdepold/feedrapp/pull/48

**Comment by sdepold** [comment]
Tada:

```
jQuery("#rss-feeds").empty().rss("https://asijnews.com/feed/", { 
  entryTemplate: "<img src='{img}'>", 
  tokens: {
    img: function (entry, tokens) { return entry.thumbnail; }
  }, 
  limit: 10
})
```

**Comment by sdepold** [comment]
Please let me know if this works for you :)

**Comment by meancode** [comment]
@sdepold When I try this, it renders the following:

    <img src="undefined">

What am I doing wrong? I am not using WP, BTW. I just want to be able to bring in the media image.

**Comment by sdepold** [comment]
can you run this code:

```
jQuery("#rss-feeds").empty().rss("https://asijnews.com/feed/", { 
  entryTemplate: "<img src='{img}'>", 
  tokens: {
    img: function (entry, tokens) { console.log(JSON.stringify(entry, null, 2)); return entry.thumbnail; }
  }, 
  limit: 10
})```

**Comment by meancode** [comment]
@sdepold You are correct, we are not running our own feedrapp. We may do that in the future, not sure.

Okay, it brought in the image using that feed. I will paste one of the JSON responses below.

Question: Is it only looking for media:thumbnail? Or does it account for other xml tags? Such as:

    <media:content url="https://static01.nyt.com/images/2016/09/13/us/13FD-TRUMP-alt/13FD-TRUMP-1473720822075-moth.jpg" medium="image" height="151" width="151"/>

Question 2: What is the best way to limit the text that is displayed from bodyPlain?

Thanks so much for the response!

Here is the JSON:

    {
      "title": "Welcome Back to School",
      "link": "https://asijnews.com/2016/09/07/welcome-back-to-school/",
      "content": "With a typhoon fast approaching Japan, it looked like the first day of school for all divisions on Wednesday, Aug 31 might have been rained out. Luckily for all, the typhoon headed up the coast and students arrived to campus for a sunny start. For many, the first chance to catch up with friends, or make new &#8230; <a href=\"https://asijnews.com/2016/09/07/welcome-back-to-school/\" class=\"more-link\">Continue reading <span class=\"screen-reader-text\">Welcome Back to&#160;School</span></a><img alt=\"\" border=\"0\" src=\"https://pixel.wp.com/b.gif?host=asijnews.com&#038;blog=43943841&#038;post=9901&#038;subd=asijnews&#038;ref=&#038;feed=1\" width=\"1\" height=\"1\" />",
      "contentSnippet": "With a typhoon fast approaching Japan, it looked like the first day of school for all divisions on Wednesday, Aug 31 mig",
      "publishedDate": "2016-09-07T05:37:00.000Z",
      "categories": [
        {
          "name": "Elementary School"
        },
        {
          "name": "High School"
        },
        {
          "name": "Middle School"
        },
        {
          "name": "Buses"
        },
        {
          "name": "First Day of School"
        },
        {
          "name": "Student Perspective"
        },
        {
          "name": "Students"
        }
      ],
      "author": "asijnews",
      "thumbnail": "https://asijnews.files.wordpress.com/2016/09/0y3a0987.jpg"
    }

**Comment by meancode** [comment]
@sdepold nevermind on the truncated body.
